### ISSUE TITLE
#[time]-decorated functions are flagged about not having documentation

### ISSUE THREAD

**Issue by arxanas** [PhilipDaniels/logging_timer#2]
If you use `warn(missing_docs)` in your project, then attempting to use `#[time]` or `#[stime]` on a `pub` function results in this warning:

```
   Compiling logging_timer_issue v0.1.0 (/Users/wkhan/workspace/logging_timer_issue)
warning: missing documentation for a function
 --> src/main.rs:8:1
  |
8 | #[stime]
  | ^^^^^^^^
  |
note: the lint level is defined here
 --> src/main.rs:3:9
  |
3 | #![warn(missing_docs)]
  |         ^^^^^^^^^^^^
  = note: this warning originates in an attribute macro (in Nightly builds, run with -Z macro-backtrace for more info)

warning: 1 warning emitted

    Finished dev [unoptimized + debuginfo] target(s) in 0.31s
```

Repro code:

```
//! This crate has documentation.

#![warn(missing_docs)]

use logging_timer::stime;

/// This function has documentation, but gets flagged anyways!
#[stime]
pub fn time_me() {
    println!("Hello, world!");
}

fn main() {
    time_me();
}
```

**Comment by PhilipDaniels** [comment]
Thanks for the report. Sorry for not replying sooner, I have been taking a break from the computer to recharge. This will because the proc macro doesn't capture the comment ... I am not sure how to fix it as I know very little about proc macros and put mine together by just copying and pasting and hacking...but I will try and take a look when I get chance.

**Comment by PhilipDaniels** [comment]
This is now fixed, I have published a new version 1.1.0. The issue actually resulted in all function attributes being stripped off, not just doc comments. Now these are preserved correctly. For example, given

```rs
#[inline]
#[stime]
#[allow(non_camel_case_types)]
/// This is a comment on foo
#[cfg(target_os = "linux")]
fn foo() {
}
```

The new expansion is

```rs
#[inline]
#[allow(non_camel_case_types)]
/// This is a comment on foo
#[cfg(target_os = "linux")]
fn foo() {
}

```
